### DISCUSSION TITLE
CVE-Search

### DISCUSSION THREAD

**Discussion by dbarzin** [dbarzin/mercator#146] [Announcements]
It is now possible with Mercator to search for CVE with the [CVE-Search](https://github.com/cve-search/cve-search) projet !
See documentation page [here](https://dbarzin.github.io/mercator/admin/#gestion-des-cve).

**Comment by MCF241** [comment]
Bonjour,

Pour les utilisateur de PostgreSQL comme moi, il se peut, parfois, que l'import des CVE bloque.
J'ai donc créé un DUMP Postgres pour faciliter l'import mais il fait quelque Mo de plus que l'original car non minimisé (67Mo au lieu de 27Mo pour MySQL).

**Comment by dbarzin** [comment]
Je suppose que tu parles de la base des données des CPE ?
Le fichier DUMP compressé avec zip fait quelle taille ?

**Reply by MCF241** [reply]
Désolé, je ne suis pas clair, il s'agit bien des CPE.
Le fichier est beaucoup plus léger en zip (6,4 Mo).

**Reply by dbarzin** [reply]
Si tu penses que cela peut être utile à d'autres utilisateurs, tu peux mettre à jour la documentation, et faire un pull request avec le fichier.
